### ISSUE TITLE
Regarding CNN implementation

### ISSUE THREAD

**Issue by ShyamSubramanian** [brmson/Sentence-selection#5]
I took a look at the paper and want to implement its CNN version. I am not sure how CNN is trained. CNN output seems to be a vector representation of Sentence. But I am not sure what the loss function is. Can you explain?
